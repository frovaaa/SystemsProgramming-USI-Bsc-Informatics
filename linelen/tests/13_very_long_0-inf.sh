#!/bin/bash
exec "$1" < tests/input1000-5002.txt
