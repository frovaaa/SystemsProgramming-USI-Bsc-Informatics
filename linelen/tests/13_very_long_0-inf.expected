1000--------------------------------------------------------...
1001++++++++++++++++++++++++++++++++++++++++++++++++++++++++...
1002========================================================...
2000--------------------------------------------------------...
2001++++++++++++++++++++++++++++++++++++++++++++++++++++++++...
2002========================================================...
5000--------------------------------------------------------...
5001++++++++++++++++++++++++++++++++++++++++++++++++++++++++...
5002========================================================...
